scripts/install.sh

```diff
@@ -15,18 +15,30 @@
 ######################################################################
 
 SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
-MODULEDIR="${SCRIPTDIR}/../helm"
+MODULEDIR="${SCRIPTDIR}/../"
 PROG="${0}"
 MODE="install"
 NS="dell-replication-controller"
 RELEASE="replication"
+MODULE="csm-replication"
 
 # export the name of the debug log, so child processes will see it
 export DEBUGLOG="${SCRIPTDIR}/install-debug.log"
-MODULE="csm-replication"
 
 source "$SCRIPTDIR"/common.sh
 
+if [ ! -d "$MODULEDIR/helm-charts" ]; then
+  if  [ ! -d "$SCRIPTDIR/helm-charts" ]; then
+    git clone --quiet -c advice.detachedHead=false -b csm-replication-1.5.0 https://github.com/dell/helm-charts
```

Comment:
Having the branch as a variable at the top of the file makes it easier to update. We can defer any changes to the future so no need to change now.